config/amarok.json

```diff
@@ -55,6 +55,10 @@
     "connextHandler": "0x0000000000000000000000000000000000000000",
     "domain": "0"
   },
+  "linea": {
```

Comment:
why do we add this if Amarok is not even deployed in this network?

Reply:
To get rid of the warning in the deploy scripts.

Reply:
Well....it doesnt make sense that the deploy script warns you for not having information about a facet that it's not even planning to deploy. I wasnt aware that it does that.. 

If that is indeed the case then I would suggest we adjust the script/warnings instead of adding meaningless and misleading config information into our repo....

Reply:
The warning is for the Receiver deploy script. It needs these values in order to even deploy.

Reply:
We can discuss further at the SC sync this week.